templateSvgIcon.ts

```diff
@@ -0,0 +1,14 @@
+const template = (variables, { tpl }) => {
+  const componentName = variables.componentName.replace('Svg', '');
+  variables.componentName = componentName;
+  variables.exports[0].declaration.name = componentName;
+
+  return tpl`
+    import { createElement } from 'react';
+    import SvgIcon from '@mui/material/SvgIcon';
+    const ${variables.componentName} = (props) => createElement(SvgIcon, props, ${variables.jsx.children});
```

Comment:
Got it working with props typing like this:

```suggestion
    const ${variables.componentName} = (props\: any) => createElement(SvgIcon, props, ${variables.jsx.children});
```

You would need to also and `--typescript` flag to the command like this:
```
npx @svgr/cli --typescript --template templateSvgIcon.ts -d src/icons --ext tsx path/to/your/file.svg

Reply:
Hey nice that you had time to look at this! Seems like the --typescript does the magic, thanks a lot!

I tested this out and seems like the `\` is unnecessary there😄

Reply:
Even better 😄